### PR TITLE
Disabling the test_CSVIter for now

### PR DESCRIPTION
**Pull request by gautamkmr**
This test causing random failure while running on windows.
Disabling it for now till we fix it. An git hub issue has
been created to track it.

@nswamy @cjolivier01

### CODE REVIEW

File: tests/python/unittest/test_io.py [hunk 1]
```diff
@@ -255,7 +257,7 @@ def check_libSVMIter_news_data():
 
     check_libSVMIter_synthetic()
     check_libSVMIter_news_data()
-
+@unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/7626")
```
Comment (eric-haibin-lin) [inline tests/python/unittest/test_io.py]: nit: Could you put the right link to the issue?

Comment (rahul003) [reply tests/python/unittest/test_io.py]: I would say that is more than a nit :)

Comment (rahul003) [reply tests/python/unittest/test_io.py]: nit here: please add a new line this line

Comment (gautamkmr) [reply tests/python/unittest/test_io.py]: Took care of the comment.

**Pull request merged**